jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/feedback/SnackbarTests.java

```diff
@@ -130,7 +130,7 @@ public void consecutiveSnackbarsTest() {
         snackbarPopUpText.is().notVisible();
 
         otherButtons.get(2).click();
-        timer.wait(() -> closePopUpSnackbarButton.waitFor().visible());
+        timer.wait(() -> closePopUpSnackbarButton.waitFor().displayed());
```

Comment:
I ran tests on branch 2713-material-snackbar, test consecutiveSnackbarsTest is still failing